Tests/Particles/ManagedCUDADeposition/make.out

```diff
@@ -0,0 +1,14 @@
+Loading ../../..//Tools/GNUMake/comps/pgi.mak...
```

Comment:
Another approach which would make the particle code a little simpler would be to use your custom allocator always, but switch between malloc/new and cudaMalloc/cudaMallocManaged (inside AMReX_CudaManagedAllocator.H) depending on whether CUDA is defined.